Dockerfile-grocy

```diff
@@ -20,7 +31,7 @@ RUN     apk update && \
         wget https://getcomposer.org/installer -O - -q | php -- --quiet && \
         pip install lastversion==0.2.4 && \
         mkdir -p /tmp/download && \
-        wget -t 3 -T 30 -nv -O "grocy.tar.gz" $(lastversion --source grocy/grocy) && \
+        wget --header "Authorization: ${GITHUB_API_TOKEN}" -t 3 -T 30 -nv -O "grocy.tar.gz" $(GITHUB_API_TOKEN=${GITHUB_API_TOKEN} lastversion --source grocy/grocy) && \
```

Comment:
Note that GitHub _does_ gracefully handle an empty value in the `Authorization` header field.  `wget --header "Authorization: " ...` is a little sloppy but does function correctly.